sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyAttributes.cs

```diff
@@ -35,9 +39,10 @@ internal struct KeyAttributes
         public string RecoveryLevel { get; internal set; }
 
         internal bool ShouldSerialize =>
-            Enabled.HasValue &&
-            NotBefore.HasValue &&
-            ExpiresOn.HasValue;
+            Enabled.HasValue ||
+            NotBefore.HasValue ||
+            ExpiresOn.HasValue ||
+            Exportable.HasValue;
```

Comment:
why did this change from `&&` to `||` ?

Reply:
Because, sadly, it was wrong before. The original idea was to consolidate decisions into the class itself rather than change the callers to decide if this nested object should be serialized. It wasn't until I added `Exportable` to the mix and when I reviewed the recordings that I saw it wasn't being recorded. While I know I exercise the other properties in other tests, they were still serialized - probably had all of them set.